lms/djangoapps/course_home_api/dates/v1/tests/test_views.py

```diff
@@ -42,6 +42,8 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
     @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
     @override_waffle_flag(COURSE_HOME_MICROFRONTEND_DATES_TAB, active=True)
     def test_get_authenticated_user_not_enrolled(self):
+        CourseEnrollment.enroll(self.user, self.course.id)
+        CourseEnrollment.unenroll(self.user, self.course.id)
```

Comment:
Are you adding these lines to all of the `_not_enrolled` tests? What happens if an enrollment actually doesn't exist? Feels like we probably want to test both routes (that might already be covered?).

Reply:
true I can test both scenarios